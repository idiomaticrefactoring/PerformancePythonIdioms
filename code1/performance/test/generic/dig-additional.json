[{"id":12149,"opcode":"QUERY","status":"NOERROR","flags":["qr","rd"],"query_num":1,"answer_num":0,"authority_num":4,"additional_num":9,"opt_pseudosection":{"edns":{"version":0,"flags":[],"udp":4096}},"question":{"name":"google.com.","class":"IN","type":"NS"},"authority":[{"name":"google.com.","class":"IN","type":"NS","ttl":172800,"data":"ns2.google.com."},{"name":"google.com.","class":"IN","type":"NS","ttl":172800,"data":"ns1.google.com."},{"name":"google.com.","class":"IN","type":"NS","ttl":172800,"data":"ns3.google.com."},{"name":"google.com.","class":"IN","type":"NS","ttl":172800,"data":"ns4.google.com."}],"additional":[{"name":"ns2.google.com.","class":"IN","type":"AAAA","ttl":172800,"data":"2001:4860:4802:34::a"},{"name":"ns2.google.com.","class":"IN","type":"A","ttl":172800,"data":"216.239.34.10"},{"name":"ns1.google.com.","class":"IN","type":"AAAA","ttl":172800,"data":"2001:4860:4802:32::a"},{"name":"ns1.google.com.","class":"IN","type":"A","ttl":172800,"data":"216.239.32.10"},{"name":"ns3.google.com.","class":"IN","type":"AAAA","ttl":172800,"data":"2001:4860:4802:36::a"},{"name":"ns3.google.com.","class":"IN","type":"A","ttl":172800,"data":"216.239.36.10"},{"name":"ns4.google.com.","class":"IN","type":"AAAA","ttl":172800,"data":"2001:4860:4802:38::a"},{"name":"ns4.google.com.","class":"IN","type":"A","ttl":172800,"data":"216.239.38.10"}],"query_time":68,"server":"192.5.6.30#53(192.5.6.30)","when":"Mon Sep 03 19:23:20 EST 2018","rcvd":287,"when_epoch":1536027800,"when_epoch_utc":null}]
