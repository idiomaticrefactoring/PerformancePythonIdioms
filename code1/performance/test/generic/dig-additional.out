; <<>> DiG 9.11.3-1ubuntu1.1-Ubuntu <<>> google.com NS @a.gtld-servers.net
;; global options: +cmd
;; Got answer:
;; ->>HEADER<<- opcode: QUERY, status: NOERROR, id: 12149
;; flags: qr rd; QUERY: 1, ANSWER: 0, AUTHORITY: 4, ADDITIONAL: 9
;; WARNING: recursion requested but not available

;; OPT PSEUDOSECTION:
; EDNS: version: 0, flags:; udp: 4096
;; QUESTION SECTION:
;google.com.            IN  NS

;; AUTHORITY SECTION:
google.com.     172800  IN  NS  ns2.google.com.
google.com.     172800  IN  NS  ns1.google.com.
google.com.     172800  IN  NS  ns3.google.com.
google.com.     172800  IN  NS  ns4.google.com.

;; ADDITIONAL SECTION:
ns2.google.com.     172800  IN  AAAA    2001:4860:4802:34::a
ns2.google.com.     172800  IN  A   216.239.34.10
ns1.google.com.     172800  IN  AAAA    2001:4860:4802:32::a
ns1.google.com.     172800  IN  A   216.239.32.10
ns3.google.com.     172800  IN  AAAA    2001:4860:4802:36::a
ns3.google.com.     172800  IN  A   216.239.36.10
ns4.google.com.     172800  IN  AAAA    2001:4860:4802:38::a
ns4.google.com.     172800  IN  A   216.239.38.10

;; Query time: 68 msec
;; SERVER: 192.5.6.30#53(192.5.6.30)
;; WHEN: Mon Sep 03 19:23:20 EST 2018
;; MSG SIZE  rcvd: 287
